metadata-store-directory-common/src/main/java/org/apache/helix/msdcommon/mock/MockMetadataStoreDirectoryServer.java

```diff
@@ -103,7 +104,7 @@ public void startServer() throws IOException {
   }
 
   public void stopServer() {
-    _server.stop(0);
+    _server.stop(STOP_WAIT_SEC);
```

Comment:
Is this timeout meaning wait 10 seconds and try stop or totally timeout is 10 second can be terminated anytime during the middle as long as it is free?

Reply:
* @param delay the maximum time in seconds to wait until exchanges have finished.
I think it is the wait time and usually returns earlier.